lib/unsplash/photo.rb

```diff
@@ -52,17 +52,19 @@ def find(id)
       # @param user [String] Limit selection to given User's ID.
       # @param query [String] Limit selection to given search query.
       # @param orientation [String] Filter by orientation of the photo. Valid values are landscape, portrait, and squarish.
+      # @param content_filter [String] Limit results by content_filter to avoid content that may be unsuitable for younger audiences. Valid values are low, high. Defaults to low.
       # @return [Unsplash::Photo] An Unsplash Photo if count parameter is omitted
       # @return [Array] An array of Unsplash Photos if the count parameter is specified. An array is returned even if count is 1
-      def random(count: nil, collections: nil, featured: nil, user: nil, query: nil, orientation: nil)
+      def random(count: nil, collections: nil, featured: nil, user: nil, query: nil, orientation: nil, content_filter: 'low')
```

Comment:
```suggestion
      def random(count: nil, collections: nil, featured: nil, user: nil, query: nil, orientation: nil, content_filter: "low")
```

Sorry didn't notice this before, prefer double quotes for strings.

Reply:
@internethostage - Changes done as per review.